cobalt/black_box_tests/testdata/scroll.html

```diff
@@ -35,18 +34,109 @@
 <body>
   <div class="app" style="overflow:auto;">
     <div id="row" class="row" style="height: 20rem;">
-      <div id="top_one" class="tile">One</div>
+      <div id="tile" class="tile">One</div>
     </div>
   </div>
 </body>
 <script>
+  // Fail if the test is not finished within 15 seconds.
+  var kTimeout = 30 * 1000;
+  var failTimer = setTimeout(fail, kTimeout);
+
+  function fail() {
+    console.log("Failing due to timeout!");
+    assertTrue(false);
+  }
+
+  function phasename(phase) {
+    switch (phase) {
+      case 0: return 'none';
+      case 1: return 'capturing';
+      case 2: return 'at target';
+      case 3: return 'bubbling';
+    }
+    return ' [unknown: ' + phase + ']';
+  }
+
+  const expected_events = [
+    // name, id, phase
```

Comment:
I think objects with named fields would be a little cleaner (and lend themselves to destructuring, assuming cobalt can handle that)

Reply:
Just to comfirm you mean something like: `[{name: 'pointermove', id: 'row', phase: Event.AT_TARGET}]` or using their own value as the field name `[{pointermove: 'pointermove', row: 'row', [Event.AT_TARGET]: Event.AT_TARGET}]`?

Reply:
The former